src/design-tokens/tier-1-definitions/typography.json

```diff
@@ -59,6 +59,139 @@
       "sm": {
         "value": "0.5px"
       }
+    },
+    "typography": {
+      "preset": {
+        "001": {
+          "@": {
+            "value": "var(--eds-font-weight-medium) var(--eds-font-size-40)/1.2 var(--eds-font-family-primary)"
+          },
+          "light": {
+            "value": "var(--eds-font-weight-light) var(--eds-font-size-40)/1.2 var(--eds-font-family-primary)"
+          },
+          "bold": {
+            "value": "var(--eds-font-weight-bold) var(--eds-font-size-40)/1.2 var(--eds-font-family-primary)"
+          },
+          "mobile": {
+            "value": "var(--eds-font-weight-bold) var(--eds-font-size-32)/1.25 var(--eds-font-family-primary)"
```

Comment:
Now that this is separated, the component itself would have to handle breaks?

Reply:
correct. I marked a TODO for this, but I think that's probably the better way to do this anyway. Tokens shouldn't have logic like that IMO

Reply:
Makes sense to me